doc/user_guide/user_guide.md

```diff
@@ -101,12 +101,18 @@ deployment script below with the desired version. (see GitHub Releases
       --bucketfs-port <BUCKETFS_PORT> \
       --bucketfs-user <BUCKETFS_USER> \
       --bucketfs-password <BUCKETFS_PASSWORD> \
+      --bucketfs_use-https <USE_HTTPS_BOOL> \
       --bucket <BUCKETFS_NAME> \
       --path-in-bucket <PATH_IN_BUCKET> \
       --language-alias <LANGUAGE_ALIAS> \ 
-      --version <RELEASE_VERSION>       
+      --version <RELEASE_VERSION> \
+      --ssl_cert_path <SSL_CERT_PATH> \
+      --use_ssl_cert_validation <USE_CERT_VALIDATION_BOOL>
```

Comment:
A common pattern for bool CLI is `--ssl-cert-validation` and `--no-ssl-cert-valiation` without a value. Click can do this.